src/plugins_types_bits.c

```diff
@@ -22,6 +22,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <stdbool.h>
```

Comment:
We are not using this header, there is `ly_bool` type defined for this purpose (with values `0` and `1`). There are some compatibility problems with `stdbool`.